src/d3a/d3a_core/export.py

```diff
@@ -384,7 +387,52 @@ def _plot_ess_soc_history(self, storage_list: list, subdir: str, root_name: str)
         plot_dir = os.path.join(self.plot_dir, subdir)
         mkdir_from_str(plot_dir)
         output_file = os.path.join(plot_dir, 'ess_soc_history_{}.html'.format(root_name))
-        BarGraph.plot_bar_graph(barmode, title, xtitle, ytitle, data, output_file)
+        PlotlyGraph.plot_bar_graph(barmode, title, xtitle, ytitle, data, output_file)
+
+    def supply_demand_curve(self, area, subdir):
```

Comment:
Ok, now there are two function called `supply_demand_curve` in this file. How about renaming this to `plot_supply_demand_curve`, similar to the other function in this class

Reply:
I followed the old convention here.
https://github.com/gridsingularity/d3a/blob/master/src/d3a/d3a_core/export.py#L273
https://github.com/gridsingularity/d3a/blob/master/src/d3a/d3a_core/export.py#L363
https://github.com/gridsingularity/d3a/blob/master/src/d3a/d3a_core/export.py#L403

Reply:
Sorry, I dont get it. Can we please discuss this offline?